app/views/owners/_form.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= form_for(@owner) do |f| %>
+  <%= f.label  :full_name %><br>
+  <%= f.text_field :first_name, placeholder: "First name..." %>
+  <!-- < f.label  :middle_name %> -->
+  <%= f.text_field :middle_name, placeholder: "Middle name..." %>
+  <!-- < f.label  :last_name %> -->
+  <%= f.text_field :last_name, placeholder: "Last name..." %><br>
+  <%= f.label  :address %><br>
+  <%= f.text_field :address, placeholder: "Street name/number..."%>
+  <!-- < f.label  :city %> -->
+  <%= f.text_field :city, placeholder: "City..."%>
+  <!-- < f.label  :state %> -->
+  <%= f.text_field :state, placeholder: "State..." %>
+  <!-- < f.label  :zip %> -->
+  <%= f.text_field :zip, placeholder: "Zipcode..."%><br>
+  <%= f.label  :years_lived_at_home %>
+  <%= f.text_field :year, class: "form-control" %>
+  <%= f.label  :email %>
+  <%= f.text_field :email, class: "form-control" %>
+  <%= f.label  :phone %>
+  <%= f.text_field :phone, class: "form-control" %>
+  <%= f.label  :Emergency_contact %><br>
+  <%= f.text_field :em_contact_name, placeholder: "Name..." %>
+  <!-- < f.label  :em_contact_phone %> -->
+  <%= f.text_field :em_contact_phone, placeholder: "Phone..." %>
+  <!-- < f.label  :em_contact_relationship %> -->
```

Comment:
Similar to the comment above, you can use labels for each field and just use text strings to update the display of the labels.

Reply:
I didn't use `labels` for every `emergency` related fields just squeeze the form at least for now. And, same for the `address`. But I'll still need to work on `text-field` styling!